job_dsl/release.groovy

```diff
@@ -4,7 +4,27 @@ common.globalWraps(){
     stage("Configure Git"){
       common.configure_git()
     }
-
+    stage("Verify component metadata"){
+      common.clone_with_pr_refs(
+        "${env.WORKSPACE}/${env.RE_JOB_REPO_NAME}",
+      )
+      venv = "${WORKSPACE}/.componentvenv"
```

Comment:
I've seen this venv creation code in a couple of places, perhaps it could go in common?